VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/non-host-waiting-room/non-host-waiting-room.component.html

```diff
@@ -0,0 +1,207 @@
+  (feedToggle)="switchStreamWindows()"
+>
+</app-video-call>
+
+<app-transfer-message *ngIf="isParticipant && isTransferringIn" [outgoingStream]="outgoingStream"></app-transfer-message>
+
+<div *ngIf="!loadingData && hearing && showExtraContent" class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <div style="margin-bottom: 0">
+      <h1 class="govuk-heading-xl">{{ 'waiting-room.your-video-hearing' | translate }} {{ getConferenceStatusText() }}</h1>
+    </div>
+
+    <app-feedback-banner *ngIf="hearing.isClosed()"></app-feedback-banner>
+
+    <div class="govuk-grid-row govuk-!-margin-bottom-4" *ngIf="!showVideo">
+      <div class="govuk-grid-column-full">
+        <div class="govuk-button-group">
+          <button id="changeCameraButton" class="govuk-button" data-module="govuk-button" type="button" (click)="showChooseCameraDialog()">
+            {{ 'waiting-room.choose-camera-and-microphone' | translate }}
+          </button>
+          <button
+            id="leaveHearingButton"
+            class="govuk-button govuk-button--warning"
+            *ngIf="isParticipant"
+            data-module="govuk-button"
+            type="button"
+            (click)="onLeaveHearingButtonClicked()"
+          >
+            {{ 'waiting-room.leave-hearing' | translate }}
+          </button>
+        </div>
+      </div>
+      <div class="govuk-grid-column-two-thirds" style="float: right">
+        <div *ngIf="isSupportedBrowserForNetworkHealth" id="hearing-monitor-container">
+          <app-participant-network-health [participant]="vhParticipant" [showDetail]="false"> </app-participant-network-health>
+        </div>
+      </div>
+    </div>
+
+    <app-hearing-details [conference]="vhConference" [hearing]="hearing"></app-hearing-details>
+  </div>
+
+  <div class="govuk-grid-column-two-thirds" *ngIf="showExtraContent">
+    <app-wait-for-hearing-panel
+      [hearing]="hearing"
+      [currentTime]="currentTime"
+      [userRole]="userRole"
+      [isQuickLinkUser]="isQuickLinkUser"
+      [isWitnessOrHasWitnessLink]="isOrHasWitnessLink()"
+    >
+    </app-wait-for-hearing-panel>
+    <ng-container *ngIf="isParticipant">
+      <div *ngIf="canStartJoinConsultation">
+        <button
+          [disabled]="!connected"
+          class="govuk-button govuk-!-margin-right-3"
+          id="openStartPCButton"
+          data-module="govuk-button"
+          (click)="openStartConsultationModal()"
+        >
+          {{ 'waiting-room.private-consultation.start' | translate }}
+        </button>
+
+        <button
+          [disabled]="!connected"
+          class="govuk-button govuk-button-blue"
+          id="openJoinPCButton"
+          data-module="govuk-button"
+          (click)="openJoinConsultationModal()"
+        >
+          {{ 'waiting-room.private-consultation.join' | translate }}
+        </button>
+      </div>
+      <app-wait-for-connection-message *ngIf="!connected"></app-wait-for-connection-message>
```

Comment:
can this sit in the parent element, right after <app-wait-for-hearing-panel> since the condition is the same for both roles

Reply:
I moved it up like I suggested and it was still the same exp. worth implementing to reduce code dupe